In [1]:
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

In [2]:
# загружаем набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - рукопашный бой (не огнестрельное оружие)
df1 = pd.read_csv('кол_во_атакующих_стратифицированная_случайная_выборка_18_инцидентов_рукопашный_бой.csv')

In [3]:
# загружаем набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - оборудование для саботажа
df2 = pd.read_csv('кол_во_атакующих_стратифицированная_случайная_выборка_18_инцидентов_саботаж.csv')

In [4]:
#набор данных грязный - имеет пропуски, надо его очистить
# -99 означает, что количество атакующих неизвестно
# пропуску тоже стоит убрать
df1['nperps']

0      5.0
1     20.0
2      3.0
3     70.0
4      4.0
5     60.0
6     28.0
7     12.0
8      1.0
9     15.0
10    30.0
11     3.0
12    14.0
13     8.0
14    40.0
15    20.0
16    30.0
17     1.0
Name: nperps, dtype: float64

In [5]:
#набор данных грязный - имеет пропуски, надо его очистить
# -99 означает, что количество атакующих неизвестно
# пропуску тоже стоит убрать
df2['nperps']

0       4.0
1       1.0
2       1.0
3       4.0
4       2.0
5     100.0
6     100.0
7       1.0
8       3.0
9      80.0
10      1.0
11     10.0
12    100.0
13     30.0
14      4.0
15      1.0
16     25.0
17      1.0
Name: nperps, dtype: float64

In [6]:
# размер набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - рукопашный бой (не огнестрельное оружие)
df1.shape

(18, 136)

In [7]:
# размер набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - саботажное оборудование
df2.shape

(18, 136)

In [9]:
# описательная статистика чистый набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - рукопашный бой (не огнестрельное оружие)
df1["nperps"].describe()

count    18.000000
mean     20.222222
std      20.019272
min       1.000000
25%       4.250000
50%      14.500000
75%      29.500000
max      70.000000
Name: nperps, dtype: float64

In [10]:
# описательная статистика чистый набор данных инцидентов на объекты инфраструктуры с использованием типа оружия - рукопашный бой (не огнестрельное оружие)
df2["nperps"].describe()

count     18.000000
mean      26.000000
std       39.050495
min        1.000000
25%        1.000000
50%        4.000000
75%       28.750000
max      100.000000
Name: nperps, dtype: float64

### Среднее значение "количества нападающих" во время атаки на инфраструктуру с использованием рукопашного боя: 20.222222 или примерно 21, так как не можем быть дробное число

### Среднее значение "количества нападающих" во время атаки на инфраструктуру с использованием оборудования для саботажа: 26